crates/weaver_forge/src/formats/markdown.rs

```diff
@@ -102,17 +134,78 @@ impl RenderContext {
         self.skip_line_prefix_on_first_line = false;
     }
 
+    fn start_unbreakable_block(&mut self, text: &str) {
+        // TODO - check for existing unbreakable.
+        if let Some(buf) = self.unbreakable_buffer.as_ref() {
+            // ToDo - we should error out here.
+            // For now, we just FLUSH this to write to the buffer.
+            let _ = self
```

Comment:
`add_blank_line` ?

Reply:
Good question - when this happens our algorithm is basically broken.  I debated between "panic"-ing and implementing.

Regarding blank lines, we have no idea what we've been buffering or why we need to buffer twice, so I think it's unlikely we have a necessary blank line, it's more likely we somehow have nested code + strong/emphasis blocks.